src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,17 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo" key={post.id}>
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      {post.userId && <UserInfo userId={post.userId} />}
+    </div>
+
+    <p className="PostInfo__body">{post.body}</p>
+
+    <CommentList postId={post.id} />
```

Comment:
if no comments => 'no comments' 
CommentList should render only list, without conditions

Reply:
as you have comments field already so pass comments as a prop to list